examples~/multiplicity/client/Program.cs

```diff
@@ -0,0 +1,526 @@
+﻿// Multiplicity Test Client
+// This test adds several dogs to the Multiplicity Test Server,
+// stores a local model of those dogs,
+// creates several connections in different combinations,
```

Comment:
Thanks for the test!

```suggestion
// creates several subscriptions in different combinations,
```